app/services/analytics_events.rb

```diff
@@ -2766,6 +2766,19 @@ def idv_in_person_usps_proofing_results_job_enrollment_updated(
     )
   end
 
+  # Tracks emails that are initiated during GetUspsProofingResultsJob
+  # @param [String] email_version success, failed or failed fraud
+  def idv_in_person_usps_proofing_results_job_email_initiated(
+    email_version:,
```

Comment:
Unsure if it was prescribed in the ticket, but "version" here had me thinking something incremented like "v1", "v2". What would you think about a "email name", "email type", "email variant", or "email template" ?

Reply:
good point, i think email type is clearer than version